src/utility.c

```diff
@@ -560,7 +560,6 @@ size_t _count_args(char **args) {
     while (args[i] != NULL && i < 1000) {
         i++;
     }
-    i++;
```

Comment:
Can you modify the comment for the code so it says it doesn't include the terminator?

Reply:
And you may need to change the tests for this too.